contracts/0.4.24/Lido.sol

```diff
@@ -458,59 +472,53 @@ contract Lido is ILido, StETH, AragonApp {
         emit ELRewardsWithdrawalLimitSet(_limitPoints);
     }
 
+    function getBufferWithdrawalsReserve() public returns (uint256) {
+        return WITHDRAWAL_RESERVE_POSITION.getStorageUint256();
+    }
+
     /**
-    * @notice Updates beacon stats, collects rewards from LidoExecutionLayerRewardsVault and distributes all rewards if beacon balance increased
+    * @notice Updates accounting stats, collects EL rewards and distributes all rewards if beacon balance increased
     * @dev periodically called by the Oracle contract
-    * @param _beaconValidators number of Lido's keys in the beacon state
-    * @param _beaconBalance summarized balance of Lido-controlled keys in wei
     */
-    function handleOracleReport(uint256 _beaconValidators, uint256 _beaconBalance) external whenNotStopped {
+    function handleOracleReport(
```

Comment:
The oracle report should invoke `reportStoppedValidators`. Looks like we don't do that.

Reply:
It does. But it moved to the oracle itself to save stack in handleOracleReport. See LidoOracleNew

Reply:
Yeah, you are totally right